server/text_generation_server/layers/gptq/__init__.py

```diff
@@ -90,13 +95,15 @@ def __init__(
         quant_method: str,
         quantize: str,
         sym: bool,
+        modules_to_not_convert: Optional[List[str]],
```

Comment:
I think it's nicer to have

```suggestion
        modules_to_not_convert: List[str],
```

here and use the empty list to signify that there are no modules no to convert, because with the current signature we have two values with the same semantics (`None` and `[]`).